spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/HyperConformanceAttributes.scala

```diff
@@ -18,18 +18,17 @@ package za.co.absa.enceladus.conformance
 import za.co.absa.hyperdrive.ingestor.api.{HasComponentAttributes, PropertyMetadata}
 
 object HyperConformanceAttributes {
-  val keysPrefix = "transformer.hyperconformance"
 
   // Configuration keys expected to be set up when running Conformance as a Transformer component for Hyperdrive
-  val menasUriKey = s"$keysPrefix.menas.rest.uri"
-  val menasCredentialsFileKey = s"$keysPrefix.menas.credentials.file"
-  val menasAuthKeytabKey = s"$keysPrefix.menas.auth.keytab"
-  val datasetNameKey = s"$keysPrefix.dataset.name"
-  val datasetVersionKey = s"$keysPrefix.dataset.version"
-  val reportDateKey = s"$keysPrefix.report.date"
-  val reportVersionKey = s"$keysPrefix.report.version"
-  val eventTimestampColumnKey = s"$keysPrefix.event.timestamp.column"
-  val eventTimestampPatternKey = s"$keysPrefix.event.timestamp.pattern"
+  val menasUriKey = "menas.rest.uri"
```

Comment:
While not sure about the `[hyperconformance]` being optional, it seems the `transformer` prefix is in any case still in place?

Reply:
I see your confusion. `[hyperconformance]` is not optional. Hyperdrive needs the prefix, e.g. `transformer.[hyperconformance].` (or `transformer.asdf` or whatever the user defines) and passes only the properties with that prefix. The prefix is cut off, because it's dynamic.
From the transformer component's point of view, there is no prefix.

Reply:
Explained live.